pkg/agent/proxy/proxier.go

```diff
@@ -397,6 +442,12 @@ func (p *proxier) uninstallNodePortService(svcPort uint16, protocol binding.Prot
 	if err := p.routeClient.DeleteNodePort(p.nodePortAddresses, svcPort, protocol); err != nil {
 		return fmt.Errorf("failed to remove NodePort traffic redirecting rules: %w", err)
 	}
+	if needClearConntrackEntries(protocol) {
+		// Remove NodePort conntrack entries when protocol is UDP.
+		if err := p.routeClient.ClearConntrackEntryForService(nil, svcPort, nil, protocol); err != nil {
```

Comment:
If I'm not mistaken, this could clear all conntrack entries on the Node using that port, even it's not even service traffic, for example, when pod A accesses pod B's this port.

Reply:
Node IPs and NodePort port are used to match the conntrack items.

Reply:
but where is Node IP matched?

Reply:
Sorry, I missed that. I'll Node IPs to match conntrack items for NodePort.